### PR TITLE
installation of mongodb for linux added to readMe

### LINKED ISSUE #149: README don't specify how to install mongo db on linux
## Type of issue:
- [ ] Bug / Error
- [ ] Idea / Feature
- [x] Improvement detail

### PR DESCRIPTION
**Pull request by war-turtle**

### CODE REVIEW

**Review by monalshadi — COMMENTED** [review]
You can as well include steps for other distros of linux. :+1:

File: README.md [hunk 1]
```diff
@@ -2,83 +2,102 @@
...
-4. To connect to the MongoDB, use the **mongo.exe** shell. For that open up a command line an run *mongo*. (Ex: "C:\Program Files\MongoDB\Server\3.4\bin\mongo.exe" ).
+4. To connect to the MongoDB, use the **mongo.exe** shell. For that open up a command line an run _mongo_. (Ex: "C:\Program Files\MongoDB\Server\3.4\bin\mongo.exe" ).
 
 For more information refer the [MongoDB manual](https://docs.mongodb.com/manual/administration/install-community/) on installing MongoDB Community edtion.
 
+## On Linux
+
+-   For debian based system
+
+1. sudo apt-get update
```
Comment (matosBrent) [inline README.md]: Added commands between backtick.

File: README.md [hunk 2]
```diff
@@ -2,83 +2,102 @@
...
+   For example, you can add the following line to your shell’s rc file (e.g. ~/.bashrc):
+   `export PATH=<mongodb-install-directory>/bin:$PATH`
+   Replace <mongodb-install-directory> with the path to the extracted MongoDB archive.
 
 ## Stackle API
+
 1. `cd` in to the **stackle_api** directory.
-2. Run `npm install` to install the packages required. 
+2. Run `npm install` to install the packages required.
 3. Once all the packages are installed run `npm start`
```
Comment (matosBrent) [inline README.md]: End this line with a fullstop.

File: README.md [hunk 3]
```diff
@@ -2,83 +2,102 @@
...
 
 ## Running Tests (Protractor)
+
 1. Run `npm install -g protractor` to install protractor globally.
 2. To get started with a Selenium Server, run
-   ```
-   webdriver-manager update
-   webdriver-manager start
-   ```
+    ```
```
Comment (matosBrent) [inline README.md]: add language tag after triple backticks. 
For eg: ```sh for shell

File: README.md [hunk 4]
```diff
@@ -2,83 +2,102 @@
...
-MEAN stack is used for developing Stackle. 
- - Front-End is developed using Angular
- - Back-end is handled by NodeJS using Express
- - MongoDB is used as the databases for storing data related to stackle
+
+MEAN stack is used for developing Stackle.
+
+-   Front-End is developed using Angular
+-   Back-end is handled by NodeJS using Express
+-   MongoDB is used as the databases for storing data related to stackle
```
Comment (matosBrent) [inline README.md]: End each point with a full stop

File: README.md [hunk 5]
```diff
@@ -2,83 +2,102 @@
...
 
 # Architecture
+
 Users interact with the application through the Angular front-end which communicates with Node-API to get and post data to the MongoDB server. The front-end also communicates with GitHub API to get data related to Users, repositories and organizations. Users can login to the application through auth0 login integrated to Stackle.
 
 ![Architecture](./stackle_app/app/images/architecture.jpg)
```
Comment (matosBrent) [inline README.md]: The image src is not working. Please check

**Review by matosBrent — CHANGES_REQUESTED** [review]
Please check comments

**Comment by war-turtle** [comment]
Done Requested Changes But can't find the missing image in the whole project

**Comment by dinaelhanan** [comment]
I used a link on my PR instead of outlining all the steps since that will 
a) avoid any errors / allow users to follow more thorough distro-specific instructions
b) reduce the space taken up by the README.md file, although that isn't that big of an issue, it helps in the long run

**Review by psnmissaka — APPROVED** [review]

**Pull request closed without merging**